--- old/features/org.eclipse.equinox.executable/library/gtk/make_linux.mak	2008-07-31 16:55:10.000000000 +0000
+++ new/features/org.eclipse.equinox.executable/library/gtk/make_linux.mak	2010-04-24 12:50:49.214235652 +0000
@@ -45,7 +45,7 @@
 LIBS = -lpthread -ldl
 GTK_LIBS = -DGTK_LIB="\"libgtk-x11-2.0.so.0\"" -DGDK_LIB="\"libgdk-x11-2.0.so.0\"" -DPIXBUF_LIB="\"libgdk_pixbuf-2.0.so.0\"" -DGOBJ_LIB="\"libgobject-2.0.so.0\""
 LFLAGS = -shared -fpic -Wl,--export-dynamic 
-CFLAGS = -g -s -Wall\
+CFLAGS += -Wall\
 	-fpic \
 	-DLINUX \
 	-DMOZILLA_FIX \
@@ -95,10 +95,10 @@
 	$(CC) $(CFLAGS) -c ../eclipseNix.c -o eclipseNix.o
 
 $(EXEC): $(MAIN_OBJS) $(COMMON_OBJS)
-	$(CC) -o $(EXEC) $(MAIN_OBJS) $(COMMON_OBJS) $(LIBS)
+	$(CC) $(LDFLAGS) -o $(EXEC) $(MAIN_OBJS) $(COMMON_OBJS) $(LIBS)
 	
 $(DLL): $(DLL_OBJS) $(COMMON_OBJS)
-	$(CC) $(LFLAGS) -o $(DLL) $(DLL_OBJS) $(COMMON_OBJS) $(LIBS)
+	$(CC) $(LDFLAGS) $(LFLAGS) -o $(DLL) $(DLL_OBJS) $(COMMON_OBJS) $(LIBS)
 
 install: all
 	cp $(EXEC) $(OUTPUT_DIR)
