Author:   Cecil Curry <leycec@gmail.com>
Upstream: no
Synopsis: Force 32-bit compilation; avoid stripping binaries.
--- memtest86+-4.00/Makefile	2009-09-22 12:35:46.000000000 +1200
+++ memtest86+-4.00.patch/Makefile	2010-03-29 08:29:22.639972138 +1300
@@ -12,6 +12,8 @@
 CC=gcc
 
 CFLAGS= -Wall -march=i486 -m32 -O2 -fomit-frame-pointer -fno-builtin -ffreestanding -fPIC
+CPPFLAGS=-m32
+ASFLAGS=-m32
 
 OBJS= head.o reloc.o main.o test.o init.o lib.o patn.o screen_buffer.o \
       config.o linuxbios.o memsize.o pci.o controller.o random.o spd.o \
@@ -31,7 +33,7 @@
 	objcopy -O binary $< memtest_shared.bin
 
 memtest: memtest_shared.bin memtest.lds
-	$(LD) -s -T memtest.lds -b binary memtest_shared.bin -o $@
+	$(LD) -T memtest.lds -b binary memtest_shared.bin -o $@
 
 head.s: head.S config.h defs.h test.h
 	$(CC) -E -traditional $< -o $@
